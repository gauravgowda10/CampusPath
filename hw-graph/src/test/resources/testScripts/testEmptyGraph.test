# Tests that a graph can be constructed

# Create a graph
CreateGraph emptyGraph

# Print the nodes in the graph
ListNodes emptyGraph