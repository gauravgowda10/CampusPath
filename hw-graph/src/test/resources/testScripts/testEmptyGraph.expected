# Tests that a graph can be constructed

# Create a graph
created graph emptyGraph

# Print the nodes in the graph
emptyGraph contains: